translation/en/RedirectedResources/assets/abdata/communication/info_14/communication_off_30/translation.txt

```diff
@@ -1,553 +1,554 @@
+「あ……ううん、気にしないで。私は思いを伝えられただけでも満足……だから……」=Ah... Don't worry about it. I'm happy just being able to convey my feelings to you... that's why...
+「……繰り返しでごめん。私、キミのこと諦められないの。もう一度、お付き合いのこと、考えてくれないかな？」=... I'm sorry to bring it up again. I just can't bring myself to give up on you. Can't you think about our relationship once again?
+「え……ほ、本当？　あ……ふふっ、ありがとうっ！　こんなに嬉しいことってないわ！」=Eh... R-Really? Ah... Fufu, Thank you! There isn't anything that could make me happier!
+「平気よ。分かってたことだから……でも……これからもキミのこと、好きでいさせて」=It's fine. I understand, so it's fine... but... let me continue liking you.
+「あ、あのさ、[H名]とHしたいなって」=I-I was thinking, I want to have sex with you, [H名].
+「え？　冗談でしょ？　うそ、本気……なの？　ごめんなさい、さすがにそれは抵抗あるわね……」=Eh? Surely that's a joke? No way, are you... serious? I'm sorry. That is a bit...
+「[H名]とHしたい」=I want to have sex with you, [H名].
+「もう……キミ、押せば何とかなるって思ってるでしょう？　まぁ、実際負けちゃうんだけど……ふふっ、いいわ、しましょ？」=Jeez... You really think I'll agree to it if you keep at it for long enough? Well, you're not wrong... Fufu, Alright. Let's do it.
+「そういうのは無理よ。身体だけじゃなくて気持ちの問題もあるし……」=I'm sorry, I just can't. It's not just a matter of the body but feelings as well...
+「ここで[H名]とHしたい」=I want to have sex with you here, [H名].
+「人……いるわよ？　本気？　だったら……付き合うけど♪　そういうプレイ……私も興味あるもの」=There are... people... here. You really want to do it here? Then, I'll go along with you ♪ This kind of fetish play... I've also been interested in it.
+「エッチっていうのは、人がいないところでするものよ？　秘め事っていうでしょう？」=Isn't sex something you have in a place devoid of people? A secret act?
+「うそ、こんなところで？　でも……ふふっ、たまにはそういうスリルもいいかもね」=No way, In a place like this? But... Fufu, sometimes fetish play like this is also fine I suppose.
+「ここでなんてダメよ。もう、勢い任せなんだから」=A place like this is no good. Jeez, I'll leave it to you, I suppose.
+「あ、あのねっ……実は私、すっごくエッチに興味があるの！　だから……その……して……くれないかな？」=Ah, well, look... In all honesty, I'm actually really interested in sex! That's why... that' is... won't you... do it with me?
+「ありがとう。変な娘だって思われるかもしれないけど……こんなこと、キミにしか頼めなくて……」=Thank you. I don't know if you think of me as a strange girl because of it but... something like this, there's no one I could ask except you...
+「わわっ、そ、そうよね！　ごめんなさい。私、ちょっとどうかしてたみたい……お願い、忘れて？」=Y-You're right! I'm sorry. I don't know what came over me... Please, can you just forget about it?
+「これからお昼、一緒にどうかな？」=How about we spend the lunch break together after this?
+「ええ、いいわよ。私もキミに声掛けようと思ってたところなの」=Okay, Alright. I was just thinking of calling you out myself.
+「ごめんなさい。今日はちょっと忙しくてあまり時間がないの。また今度誘ってね」=I'm really sorry. I'm a bit busy today, so I have no time. Invite me out again sometime, alright?
+「今日のお昼は先約がいるかしら？　もしフリーなら一緒にどう？」=Do you have plans with anyone for the lunch break today? If you're free, wanna spend it together?
+「ふふっ、ありがとう。キミと一緒なら楽しいお昼が過ごせるの」=Fufu, Thank you. With you, I can pass the lunch break while having fun.
+「あら、残念……それじゃまた誘うわね」=Oh, that's too bad... Then I'll invite you again some other time.
+「俺と一緒にコイカツしない？」=Wanna do love activities with me?+B2178
```

Comment:
`+B2178`?

Reply:
some junk data got snuck in somewhere it seems, cleaned up